config/initializers/dfe-analytics.rb

```diff
@@ -0,0 +1,49 @@
+DfE::Analytics.configure do |config|
+  # Whether to log events instead of sending them to BigQuery.
+  #
+  config.log_only = false
+
+  # Whether to use ActiveJob or dispatch events immediately.
+  #
+  # config.async = true
+
+  # Which ActiveJob queue to put events on
+  #
+  config.queue = :big_query
+
+  # The name of the BigQuery table we’re writing to.
+  #
+  config.bigquery_table_name = 'events'
+
+  # The name of the BigQuery project we’re writing to.
+  #
+  config.bigquery_project_id = ENV['BIG_QUERY_PROJECT_ID']
+
+  # The name of the BigQuery dataset we're writing to.
+  #
+  config.bigquery_dataset = ENV['BIG_QUERY_DATASET']
+
+  # Service account JSON key for the BigQuery API. See
+  # https://cloud.google.com/bigquery/docs/authentication/service-account-file
+  #
+  config.bigquery_api_json_key = ENV['BIG_QUERY_API_JSON_KEY']
+
+  # Passed directly to the retries: option on the BigQuery client
+  #
+  # config.bigquery_retries = 3
```

Comment:
Since we had 3 retries in the deleted code, do we need this to be set for the new jobs? https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/workers/send_events_to_bigquery.rb#L5

Reply:
Good question. We don't actually have a "retries" value for the activejob job — maybe we should add a sensible default? wdyt?

Reply:
That makes sense, just reading on docs that sidekiq retries 25 times by default? but that seems excessive. I guess 3 or thereabouts would be more sensible

Reply:
yes that's a lot! I'll see if I can amend in gem